### PR TITLE
Add more video playback rates

### PR DESCRIPTION
**Pull request by taneliang**
A student in CS1010X requested for more playback rates. This small PR adds more playback rates at 0.25X intervals from 0.25-1.5X.

screenshot_20180122_195317

### CODE REVIEW

**Comment by taneliang** [comment]
Closing and reopening to run Travis again

File: client/app/lib/constants/videoConstants.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ export const youtubeOpts = {
 
 export const videoDefaults = {
   volume: 0.8,
-  availablePlaybackRates: [0.5, 1, 1.5, 2, 2.5],
+  availablePlaybackRates: [0.5, 1, 1.25, 1.5, 1.75, 2, 2.25, 2.5],
```
Comment (weiqingtoh) [inline client/app/lib/constants/videoConstants.js]: Shall we do the following instead `0.25, 0.5, 0.75, 1, 1.25, 1.5, 2.0, 2.5`. I think the 0.25 after 1.5x doesn't really make much of a difference, so no point including so many of those.

**Comment by benleong** [comment]
I agree with Weiqing. :-)

Good thinking!

File: client/app/lib/constants/videoConstants.js (hunk 1 above)
Comment (taneliang) [reply client/app/lib/constants/videoConstants.js]: Good idea, and it's closer to YouTube's playback rates too.

**Comment by weiqingtoh** [comment]
@taneliang can squash commits, and I'll merge after that! 👍

**Comment by taneliang** [comment]
@weiqingtoh Done! Sorry didn't realize that was expected

**Comment by weiqingtoh** [comment]
@taneliang thanks - Coursemology is quite large, so we try to keep the commit history as clean as possible so that it is easier to track why things are done in a certain way! 👍

**Pull request merged**